cscs-checks/apps/quantumespresso/quantumespresso_check.py

```diff
@@ -56,7 +55,7 @@ def __init__(self, scale, variant):
         energy = sn.extractsingle(r'!\s+total energy\s+=\s+(?P<energy>\S+) Ry',
                                   self.stdout, 'energy', float)
         energy_diff = sn.abs(energy-energy_reference)
-        self.sanity_patterns = sn.all([
+        self.sanity_patterns += sn.all([
             sn.assert_lt(energy_diff, 1e-8)
         ])
```

Comment:
This will not have the desired result and it will always succeed if the final list has at least one element. The correct way to write this is the following:

```suggestion
        self.sanity_patterns = sn.all(
            self.sanity_patterns,
            [sn.assert_lt(energy_diff, 1e-8)]
        )
```

Reply:
Thanks @vkarak, I have changed the code accordingly